cmd/kube-apiserver/app/aggregator.go

```diff
@@ -276,6 +276,8 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "coordination.k8s.io", Version: "v1"}:               {group: 16500, version: 15},
 	{Group: "coordination.k8s.io", Version: "v1beta1"}:          {group: 16500, version: 9},
 	{Group: "auditregistration.k8s.io", Version: "v1alpha1"}:    {group: 16400, version: 1},
+	{Group: "node.k8s.io", Version: "v1alpha1"}:                 {group: 16300, version: 9},
```

Comment:
make alpha=1, beta=9 to match others

Reply:
Done (although 9 is actually more common for v1alpha1, 3 vs 2).